pool/test/session_test.ml

```diff
@@ -551,14 +551,21 @@ let delete_session_with_follow_ups () =
   check_result (Error Pool_common.Message.SessionHasFollowUps) res
 ;;
 
-let create_cancellation_message reason contact =
+(* TODO: Add history? *)
```

Comment:
what about this todo?

Reply:
Removed the comment.

In my next PR I will add this feature add store the queue id if a job is resent.